src/main/scala/firrtl/Driver.scala

```diff
@@ -90,7 +92,20 @@ object Driver {
       val annotationFile = new File(annotationFileName)
       if (annotationFile.exists) {
         val annotationsYaml = io.Source.fromFile(annotationFile).getLines().mkString("\n").parseYaml
-        val annotationArray = annotationsYaml.convertTo[Array[Annotation]]
+        val annotationArray = {
+          val annos = annotationsYaml.convertTo[Array[Annotation]]
+
+          if (annos.nonEmpty) {
+            annos ++ List(Annotation(
+              ModuleName("All", CircuitName("All")),
```

Comment:
I dislike this... I assume it isn't possible to get the Circuit name here. At the very least, make it just a CircuitName, not ModuleName(..).